### ISSUE TITLE
BUG: Singel F-Key to open FLow doesnt work in some context

### ISSUE THREAD

**Issue by SchittBull** [Flow-Launcher/Flow.Launcher#1408]
### Checks

- [x] I have checked that this issue has not already been reported.

### Problem Description

I use F19 to open Flow. When some apps are focused this doesn't work. For Example when services.exe is open. It only works with alt or ctrl in combination with the F-Keys. 

### To Reproduce

1. Set Flow Launcher hotkey to F18 (or any F-Key)
2. win + r "services.msc"
3. Flow launcher doesn't open when pressing F-Key

### Flow Launcher Version

v1.9.4

### Windows Build Number

[Version 10.0.22000.978]

**Comment by taooceros** [comment]
maybe services.msc is occupying the F key? Flow didn't use a very high privilege hotkey binding for its global hotkey.

**Comment by SchittBull** [comment]
Yes probably. I already tried starting as a administrator. It seems no single keypresses are working if some windows are focused.

**Comment by TheBestPessimist** [comment]
FYI, StarCraft 2 has the same issue with FL. Whatever shortcut i set for FL (ie. <kbd>Alt</kbd>+<kbd>End</kbd> or <kbd>Win</kbd>+<kbd>F10</kbd>), it doesn't launch FL.

Not sure if related but if start menu is open, and i press the shortcut for FL, FL launches behind start and all keyboard input goes to the start instead of FL.

**Comment by TheBestPessimist** [comment]
bad bot

**Comment by TheBestPessimist** [comment]
Please add label [`keep-fresh](https://github.com/Flow-Launcher/Flow.Launcher/issues?q=is%3Aopen+label%3Akeep-fresh+sort%3Aupdated-desc)
